packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/appmesh.ts

```diff
@@ -347,7 +347,7 @@ export class AppMeshExtension extends ServiceExtension {
     // Next update the app mesh config so that the local Envoy
     // proxy on this service knows how to route traffic to
     // nodes from the other service.
-    this.virtualNode.addBackend(appmesh.Backend.virtualService({ virtualService: otherAppMesh.virtualService }));
+    this.virtualNode.addBackend(appmesh.Backend.virtualService(otherAppMesh.virtualService ));
```

Comment:
No need for the space before the `))`:

```suggestion
    this.virtualNode.addBackend(appmesh.Backend.virtualService(otherAppMesh.virtualService));
```